src/algopy_testing/op.py

```diff
@@ -1040,54 +1040,66 @@ def app_address(a: algopy.Application | algopy.UInt64 | int, /) -> tuple[algopy.
 
 class _AppLocal:
     @staticmethod
-    def get_bytes(
-        a: algopy.Account | algopy.UInt64 | int, b: algopy.Bytes | bytes, /
-    ) -> algopy.Bytes:
-        import algopy
-
+    def _get_local_state(key: str) -> algopy.LocalState[Any]:
```

Comment:
raw underlying key should be `bytes` as it is possible for global/local state to have a key that is not a UTF8 string.